application/src/command.rs

```diff
@@ -0,0 +1,55 @@
+use crate::port::{TodoListRepository, TodoListStore};
+use domain::todolist_message::TodoListMessage;
+use framework::*;
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+pub enum Command {
+    AddTask { name: String },
+    RemoveTask { index: usize },
+    CompleteTask { index: usize },
+}
```

Comment:
**Suggestion:** Consider adding a docstring to the `Command` enum. [best practice]
```suggestion
#[derive(Deserialize)]
/// Represents a command to be executed on the todo list.
pub enum Command {
```